azerothcore/azerothcore-wotlk
# fix(DB/BWL): Trash formations

## Changes Proposed:
-  Added formations

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10907

## SOURCE:


## Tests Performed:

- Not tested.

## How to Test the Changes:


1. Go to Blackwing Lair
2. See the issue for the trash to test and pull them.
3. Success ?:D
